doc/man1/openssl-s_time.pod.in

```diff
@@ -87,12 +87,6 @@ I<option> argument can be a single option or multiple options separated by
 commas.  Alternatively the B<-nameopt> switch may be used more than once to
 set multiple options. See the L<openssl-x509(1)> manual page for details.
 
-=item B<-CApath> I<directory>
```

Comment:
Why is this removed?

Reply:
Because @richsalz forgot to when introducing the man-related variables. This one is in `$OpenSSL::safe:whatitwas_trust_item`